packages/realm/src/app-services/SyncSession.ts

```diff
@@ -93,6 +109,21 @@ function toBindingDirection(direction: ProgressDirection) {
   }
 }
 
+function isEstimateProgressNotification(cb: ProgressNotificationCallback): cb is DynamicProgressNotificationCallback {
```

Comment:
1. I believe a `Callback` postfix is missing from the name.
2. Could we use the fully spelled out name `callback` for increased consistency?

Reply:
agree with the callback, these bits of code are a mix of mine and largely @kneth so he may want to weigh in as well